web-ts/src/RootSwitcher.tsx

```diff
@@ -62,22 +63,26 @@ const RootSwitcher = ({ setContentLanguage }: PropsType): ReactElement => {
       </Layout>
     )
   }
+  const relevantCities = fixedCity ? cities.filter(city => city.code === fixedCity) : cities
 
   return (
     <Switch>
-      <Route exact path={RoutePatterns[LANDING_ROUTE]} render={props => <LandingPage cities={cities} {...props} />} />
+      <Redirect exact from='/' to={fixedCity ? cityContentPath : landingPath} />
+      <Redirect exact from={`/:cityCode`} to={cityContentPath} />
```

Comment:
This redirect effectively prevents all urls with only a single url param from working (i.e. main-disclaimer, not-found) and also shadows the landing redirect. That's why I put those redirects last.